src/tinypeel/tinypeel.py

```diff
@@ -213,6 +213,107 @@ def generateSingleLocusSegregation(peelingInfo, pedigree, args):
         peelingInfo.segregation[:, :, :] = 0.25
 
 
+def get_probability_options():
+    parse_dictionary = dict()
+    parse_dictionary["geno_error_prob"] = lambda parser: parser.add_argument(
+        "-geno_error_prob",
+        default=0.0001,
+        required=False,
+        type=float,
+        help="Genotyping error rate. Default: 0.0001.",
+    )
+    parse_dictionary["seq_error_prob"] = lambda parser: parser.add_argument(
+        "-seq_error_prob",
+        default=0.001,
+        required=False,
+        type=float,
+        help="Sequencing error rate. Default: 0.001.",
+    )
+    parse_dictionary["recombination"] = lambda parser: parser.add_argument(
+        "-recomb",
+        default=1,
+        required=False,
+        type=float,
+        help="Recombination rate per chromosome (in Morgan). Default: 1.",
+    )
+    return parse_dictionary
+
+
+def get_input_options():
+    parse_dictionary = dict()
+    parse_dictionary["bfile"] = lambda parser: parser.add_argument(
+        "-plink_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A file in plink (binary) format. Only stable on Linux).",
+    )
+    parse_dictionary["genotypes"] = lambda parser: parser.add_argument(
+        "-geno_file",
+        default=None,
+        required=False,
+        type=str,
+        nargs="*",
+        help="A file in AlphaGenes format.",
+    )
+    parse_dictionary["reference"] = lambda parser: parser.add_argument(
+        "-reference",
```

Comment:
@AprilYUZhang do we mention this file at all in documentation? I think we are not. Please open an issue to sort this.